ow_plexil/src/plans/InitializeLander.plp

```diff
@@ -0,0 +1,25 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Reset pan and tilt angles to 0 and stow the arm (if not already there)
+
+#include "plan-interface.h"
+
+InitializeLander:
+{
+  InOut Boolean initialize;
+
+  log_info ("Initializing lander...");
+
+  log_info ("Resetting pan angle to 0...");
+  SynchronousCommand pan_antenna (0);
+  log_info ("Resetting tilt angle to 0...");
+  SynchronousCommand tilt_antenna (0);
+  log_info ("If arm unstowed, restowing the arm...");
```

Comment:
Stow is idempotent, so it doesn't matter if the arm was unstowed or not.   You can just say "Stowing the arm".